utils/wmlxgettext

```diff
@@ -44,23 +44,28 @@ import pywmlx
 def commandline(args):
     parser = argparse.ArgumentParser(
         description='Generate .po from WML/lua file list.',
-        usage='''wmlxgettext --domain=DOMAIN [--directory=START_PATH]
+        usage='''wmlxgettext --domain=DOMAIN -o OUTPUT_FILE
+                   [--directory=START_PATH]
                    [--recursive] [--initialdomain=INITIALDOMAIN]
                    [--package-version=PACKAGE_VERSION]
-                   [--no-text-colors] [--fuzzy] [--warnall] [-o OUTPUT_FILE]
+                   [--no-text-colors] [--fuzzy] [--warnall]
                    FILE1 FILE2 ... FILEN'''
     )
     parser.add_argument(
         '--version',
         action='version',
-        version='wmlxgettext 2016.10.03.py3'
+        version='wmlxgettext 2017.06.25.py3'
     )
     parser.add_argument(
         '-o',
+        required=True,
         default=None,
         dest='outfile',
-        help= ('Destination file. By default the output '
-               'will be printed on stdout')
+        help= ('Destination file. In some special situation you could want to '
+               'write the output to STDOUT write an actual file. In that case '
+               'you can use "-o -" to write the pot file to STDOUT on purpose '
+               '(wich it is something you normally would avoid, becouse it can '
```

Comment:
Typo: "which". Also, I assume you mean "to STDOUT instead of an actual file" rather than "write" there.

The text is also slightly awkward-sounding to a native speaker. Some possible fixes:

* Change "you could want" to "you might want" or "you may want".
* Add a comma after "In that case".
* "which is something you would normally avoid" - basically swapping two words and removing the "it"